### PR TITLE
Added solution and readme back

### PR DESCRIPTION
**Pull request by DanielBachler**
Will be redoing the readme in a bit

### CODE REVIEW

File: problems/euler/num22-names-scores/ruby/DanielBachler/solution.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Enter your code here. Read input from STDIN. Print output to STDOUT
```
Comment (Jason-Bengtson) [inline problems/euler/num22-names-scores/ruby/DanielBachler/solution.rb:1]: Maybe elaborate on this comment with what you had in the readme, with things like enter an integer, followed by that many names (was a little confused at first trying to immediately figure out what to do.

File: problems/euler/num22-names-scores/ruby/DanielBachler/solution.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    score = 0
+    0.upto word.length-1 do |i|
+        score += (word[i].ord - 64) 
+    end
+    scores[word] = score
+end
+toGet.each do |word|
+    multiplier = words.find_index(word) + 1
+    puts scores[word] * multiplier
+end
```
Comment (Jason-Bengtson) [inline problems/euler/num22-names-scores/ruby/DanielBachler/solution.rb:30]: perhaps have an output showing the task has competed, and the values so there is something to test.

**Review by Jason-Bengtson — COMMENTED** [review]
The program compiled and ran, no output made it a bit hard to test, and had a bit of trouble initially realizing it wanted me to input something as there was no text prompting me to. Otherwise the code was clean and the logic seems sound, good work!
+1
command line ruby

**Review by Jason-Bengtson — CHANGES_REQUESTED** [review]
Just change to have an input prompt, and add an output display to help in testing/knowing program is complete. Thanks!

**Comment by tyrado** [comment]
+10
I would add like a end solution or maybe a counter for the names so when you put in 4 for the amount of people have that input also go to a loop that takes it in and says like here is person 1: ..... but overall looks great

**Comment by tyrado** [comment]
solution
So i was able to give someone a number for a name and I dont think im supposed to be able to do that

**Review by Jason-Bengtson — APPROVED** [review]
runs!
+10

**Pull request merged**